### PR TITLE
Get Default Location of Event Logs from Registry

### LINKED ISSUE #785: [Issue] Exchange Log Collector - Default location of Event Logs
**Describe the issue**
Customer moved the default location of the Event Logs to a different drive. The way the script collects the data is based off the default location and need to determine the correct location on the fly. 

**Expected behavior**
Collect the data correctly based off where the events are being logged at. 

**Additional context**
Should be able to determine the location of the event logs from here in the registry. However, need to figure out where the rest of the Exchange Event Logs would be kept at. 

image

### PR DESCRIPTION
**Pull request by dpaulson45**
**Describe the issue**
Customer moved the default location of the Event Logs to a different drive. The way the script collects the data is based off the default location and need to determine the correct location on the fly. 

**Fix**
Get the appropriate location from the registry if possible. Otherwise, default back to the old logic of the hard coded path. 

Didn't find a registry location for the other Exchange Logs that we are looking for, however, doesn't mean that they aren't there. For this time only worried about these logs as these were the only ones that we have seen moved to a different location before.

Resolved #785

### CODE REVIEW

File: Diagnostics/ExchangeLogCollector/RemoteScriptBlock/IO/Save-WindowsEventLogs.ps1 [hunk 1]
```diff
@@ -14,15 +15,52 @@ Function Save-WindowsEventLogs {
...
+
+        if ($null -eq $applicationLogs) { $applicationLogs = "$rootLogPath\Application.evtx" }
+
+        if ($null -eq $systemLogs) { $systemLogs = "$rootLogPath\System.evtx" }
+
+        if ($null -eq $managementLogs) { $managementLogs = "$rootLogPath\MSExchange Management.evtx" }
+
+        $logs += $applicationLogs
+        $logs += $systemLogs
+        $logs += $managementLogs
```
Comment (bill-long) [inline Diagnostics/ExchangeLogCollector/RemoteScriptBlock/IO/Save-WindowsEventLogs.ps1:49]: The repetition here is bugging me. What about:

```powershell
foreach ($logName in @("Application", "System", "MSExchange Management")) {
  $regValue = Get-RemoteRegistryValue "$baseRegistryLocation$logName"
  $logs += ($null -ne $regValue ? $regValue : "$rootLogPath\$logName.evtx")
}
```

Comment (dpaulson45) [reply Diagnostics/ExchangeLogCollector/RemoteScriptBlock/IO/Save-WindowsEventLogs.ps1:49]: would have to change it to an `if` statement not a `?`. Only available in PowerShell Core.

Comment (dpaulson45) [reply Diagnostics/ExchangeLogCollector/RemoteScriptBlock/IO/Save-WindowsEventLogs.ps1]: how does this look now?

**Review by bill-long — APPROVED** [review]

File: Diagnostics/ExchangeLogCollector/RemoteScriptBlock/IO/Save-WindowsEventLogs.ps1 (hunk 1 above)
Comment (bill-long) [reply Diagnostics/ExchangeLogCollector/RemoteScriptBlock/IO/Save-WindowsEventLogs.ps1]: Better.

**Pull request merged**